css/properties/-webkit-line-box-contain.json

```diff
@@ -5,7 +5,8 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "12",
+              "version_removed": "47"
```

Comment:
✅ Added in Chrome 12 (https://github.com/caugner/css-properties/commit/31ff6c8ee553e428fd2d1be7b32f219ec3e6ac36) and removed in Chrome 47 (https://github.com/caugner/css-properties/commit/f3255fe38373768bc0c0e875667f26ba0cb96c33).